src/lora.cpp

```diff
@@ -0,0 +1,61 @@
+#include "lora.hpp"
+#include <ArduinoJson.h>
+#include <Base64.h>
+#include <Arduino.h>
+#include "result.hpp"
+
+
+namespace obc{
+
+HardwareSerial Serial5(PD2, PC12);
+
+Result<Unit, Errc> init_lora()
+{
+    constexpr auto baud_rate = 9600l;
+    constexpr std::array arr2 = {"AT+DR=EU868", "AT+CH=NUM,0-2", "AT+MODE=LWOTAA","AT+JOIN"};
+    Serial5.begin(baud_rate);
+
+    for (const auto elem : arr2) {
+        Serial5.println(elem);
+        delay(10000);
+        String payload = Serial5.readString();
+        if (payload.startsWith("ERR")) {
+            return Err{Errc::Busy};
+        }
+    }
+
+    return Ok{Unit{}};
+}
+
+void lora_serialize(const Packet& data){
+    constexpr auto baud_rate = 9600l;
+    Serial.begin(baud_rate);
```

Comment:
o ile sie nie myle to ten sam serial juz jest inicjalizowany gdzies indziej wiec te linijke mozna wywalic?

Reply:
yes, mamy w main deklarację, można go chyba externować tutaj i w sumie podobna sytuacja jest w gps, więc można zrobić to samo

Reply:
Ok, tutaj z tego co widzę, to ten serial jest wykorzystany tylko do sprawdzania encoded stringa, więc docelowo można go zupełnie stąd wywalić